README.md

```diff
@@ -30,25 +30,25 @@ This gem is an alternative and inspired by the great work in the other gem [simp
 Whereever you are integrating `SimpleCov` you can configure the `SimpleCovSmallBadge` gem as any formater can be configured. The default integration could looks as follows:
 
 ```ruby
-require 'simple-cov-small-badge'
+require 'simplecov_small_badge'
```

Comment:
I've tried using the "simple-cov-small-badge" name at start and it couldn't load it, but then I noticed in the spec_helper of the repo that you are using it this way:  [spec_helper.rb](https://github.com/skyfusion89/simplecov-small-badge/blob/772269f2335bda7d1e9fa42ab96275c7e91dd121/spec/spec_helper.rb#L22)